Products/CMFCore/FSPageTemplate.py

```diff
@@ -114,7 +114,10 @@ def _readFile(self, reparse):
                 if xml_info:
                     # Smells like xml
                     # set "content_type" from the XML declaration
-                    encoding = xml_info.group(1) or 'utf-8'
+                    if xml_info.group(1):
+                        encoding = xml_info.group(1).decode('ascii')
```

Comment:
Decode should only be done on Python 3 as on Python 2 encoding will become `unicode` otherwise:

```python
Python 2.7.17 (default, Oct 20 2019, 14:46:50)
Type "help", "copyright", "credits" or "license" for more information.
>>> 'utf-8'.decode('ascii')
u'utf-8'
```

Reply:
Good catch! As the test did not fail, there seems to be no consequence, but most certainly it is better to keep the previous behavior for Python 2.